Source/core/TokenizedStringList.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+#include "Module.h"
+
+namespace WPEFramework {
+namespace Core {
+
+template <char SEPARATOR, bool REMOVE_WHITESPACES>
+class TokenizedStringList {
+public:
+    TokenizedStringList() = default;
+    ~TokenizedStringList() = default;
+    TokenizedStringList(const TokenizedStringList&) = default;
+    TokenizedStringList& operator=(const TokenizedStringList&) = default;
+
+    explicit TokenizedStringList(const string& buffer)
+    {
+        Tokenize(buffer);
+    }
+
+    void Tokenize(string buffer)
+    {
+        if (!buffer.empty()) {
+            //remove spaces and tokenize by comma
+            if (REMOVE_WHITESPACES) {
+                buffer.erase(std::remove_if(buffer.begin(), buffer.end(), ::isspace), buffer.end());
+            }
+            std::istringstream iss(buffer);
+
+            string substring;
+            while (iss.good()) {
+                getline(iss, substring, SEPARATOR);
+                _parts.push_back(substring);
+            }
+        }
+    }
+
+    void Clear()
+    {
+        _parts.clear();
+    }
+
+    const string& First() const
+    {
+        return _parts.front();
+    }
+
+    string All() const
+    {
+        string result;
+
+        if (!_parts.empty()) {
+
+            for (const auto& part : _parts) {
+                result += part;
+                result += ',';
```

Comment:
I think we want the SEPARATOR here and not the ','. If you update it also hed the Copyright header notice to this file ;-)